cypress/integration/options/totals.cy.js

```diff
@@ -68,9 +69,10 @@ describe('Options - Column totals', () => {
         selectDataElements(['ART enrollment stage 1'])
         clickDimensionModalHideButton()
 
-        const year = new Date().getFullYear().toString()
+        const year = (new Date().getFullYear() - 1).toString()
```

Comment:
This is used so many times, perhaps it would make sense as a separate function `getPreviousYear` ?

Reply:
Agree, we do have that in LL.

Reply:
Done.